src/applications/caregivers/components/FormPages/FacilityConfirmation.jsx

```diff
@@ -5,35 +5,52 @@ import FormNavButtons from 'platform/forms-system/src/js/components/FormNavButto
 const FacilityConfirmation = props => {
   const { data, goBack, goForward } = props;
   const selectedFacility = data['view:plannedClinic'].veteranSelected;
+  const selectedCaregiverSupportFacility =
+    data['view:plannedClinic'].caregiverSupport;
 
-  return (
-    <div>
-      <h3>Confirm your health care facilities</h3>
-      <h4>The Veteran’s Facility you selected</h4>
-      <p>
-        This is the facility where you told us the Veteran receives or plans to
-        receive treatment.
-      </p>
-      <p className="va-address-block">
-        {selectedFacility.name}
-        {selectedFacility?.address?.physical?.address1 && (
+  const addressText = facility => {
+    return (
+      <>
+        <h5 className="vads-u-font-size--h4 vads-u-margin-top--0">
+          {facility.name}
+        </h5>
+        {facility?.address?.physical?.address1 && (
           <>
+            {facility.address.physical.address1}
             <br />
```

Comment:
For accessibility reasons, any `<br>` tags here need to have `role="presentation"` added to them. There is an odd thing with most screen readers where it will try to actually read these out as a line break. And since we're using it just for visual formatting, we don't want this when reading out an address.

Reply:
Makes sense! I updated it.